### ISSUE TITLE
Unable to focus on the nvda speech viewer when the select synthesizer window is open

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9808]
### Steps to reproduce:
Open Speech Viewer

Now open the settings of the nvda switch to talk tab and press the change button.
The window
select Synthesizer will open.
Now press alt + tab to focus in the Speech Viewer window
### Actual behavior:
or nvda informs
Speech Viewer
unavailable
And it does not allow the reading of the speech history with the arrows.
### Expected behavior:
Nvda should announce
Speech Viewer
and allow the text to be read with the arrows.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
alpha-17697,1d50498c#### 
Windows version:
10 17763.557
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by bhavyashah** [comment]
@fernando-jose-silva  I am unable to replicate this issue with NVDA version 2020.2 Beta 1. Having the Select Synthesizer dialog open has no negative impact on my experience with accessing the Speech Viewer, irrespective of whether I open the Select Synthesizer dialog before or after enabling the Speech Viewer. Given that your report is more than a year old, could you please check again and report whether you are experiencing this problem with the latest version of NVDA?

**Comment by fernando-jose-silva** [comment]
[viwer.txt](https://github.com/nvaccess/nvda/files/4808554/viwer.txt)
strange, I managed to replicate the problem.
follows log:
open the speech viewer
open nvda settings
choose the speech tab
press the change button
open the window to choose a synthesizer.
switch with alt + tabe until you hear speech viewer
You will now hear unavailable.
press tab, silence.
without add-ons.

**Comment by Adriani90** [comment]
cc: @feerrenrut

**Comment by feerrenrut** [comment]
The speech synthesizer selection window is modal (other windows can't be activated) by design, can you explain why this causes a problem for you? Knowing why you want to view the speech viewer while this window is open will help us to come up with an appropriate solution.

**Comment by fernando-jose-silva** [comment]
This call has long been open.
From what I remember when I discovered this case, I would like to capture in the speech viewer the names of the controls displayed on the screen of the synthesizer exchange window, and the names of the synthesizers as well as explanatory texts in this window to write a handout used in course we give about the use of nvda in the institution where I work.
When writing materials about nvda it is very useful to be able to capture the names of the element controls in the nvda window, and in this case also capture the name of the synthesizers that are listed in this window.
I know there are other ways to do the same thing, I just think that using the speech viewer could help.

**Comment by feerrenrut** [comment]
One way you can do in this is to first open the speech viewer, then via the speech panel open the synth selection dialog. Focus each of the controls that you want the text for, then close the synth dialog and go to the speech viewer and scroll up.

**Comment by Adriani90** [comment]
@fernando-jose-silva is the solution described in the last comment above enough to solve your problem? Or do you still have this problem?

**Comment by fernando-jose-silva** [comment]
I did a test now, and I was no longer able to reproduce this problem. Thanks.

**Comment by Adriani90** [comment]
Thanks for the updates, closing as works for me.
